components/config/src/config/taxonomies.rs

```diff
@@ -11,9 +13,12 @@ pub struct Taxonomy {
     pub paginate_path: Option<String>,
     /// Whether to generate a feed only for each taxonomy term, defaults to false
     pub feed: bool,
-    /// The language for that taxonomy, only used in multilingual sites.
-    /// Defaults to the config `default_language` if not set
-    pub lang: String,
+
+    // Only for convenience/backwards compatibility
+    #[serde(skip_deserializing)]
+    pub lang: LanguageIdentifier,
```

Comment:
that's still going to break if someone used an invalid language identifier right?

Reply:
This field [won't be read](https://serde.rs/field-attrs.html#skip_deserializing) from the config, just passed to templates. We set it from the containing `LocaleOptions`' key in `config.languages`. Any invalid language will error out when parsing the config.

I might not get what you're saying though.